app/views/subnets/index.html.erb

```diff
@@ -8,7 +8,7 @@
     <tr>
       <th><%= sort :name, :as => s_("Subnet|Name") %></th>
       <th><%= sort :network, :as => s_("Subnet|Network") %></th>
-      <th><%= _("Domains") %></th>
+      <th><%= sort :domains, :as => _("Domains") %></th>
```

Comment:
Invalid search query: the field 'domains' in the order statement is not valid field for search